runtime/compiler/optimizer/VectorAPIExpansion.hpp

```diff
@@ -144,14 +144,17 @@ class TR_VectorAPIExpansion : public TR::Optimization
    *  vector species, element type, or number of lanes
    *
    */
-   enum vapiArgType
+   enum vapiObjType
```

Comment:
The VectorAPIExpansion class defines only two enums: `handlerMode` and `vapiObjType`. I agree that the latter  is a bit confusing but was hoping it's reasonably self-explanatory from the elements of the enum: `Vector, Scalar, numLanes`, etc. Basically it lists all kinds of objects of the Vector API library that we need to recognize and handle appropriately. Perhaps calling the `enum vectorAPIObject` would be nicer? I would like to keep it short since used often. On the other hand, I think members of the two enums don't not need any special prefix.  Please let me know what you think.

Reply:
sorry, i was not clear.  i referred to the inconsistent enum or type naming.  for example, always start with a capital letter.  numLanes vs. NumLanes;  or elementType vs. ElementType; etc etc

Reply:
Oh, I see. Yeah, need to think of some nice naming so that they are not too long but also stand out and can be  easily distinguished from local variables.

Reply:
Changed numLanes to NumLanes and elementType to ElementType. Indeed looks better. Thanks for the suggestion!